### ISSUE TITLE
Format distance fields

### ISSUE THREAD

**Issue by frodrigo** [teritorio/vido#90]
Like time, have a human/i18n format for distance

> pedestre : 10 h 30 min, normal.
> 42.175 km

**Comment by wazolab** [comment]
I think it's already the case no (see the following screenshots)
I might have done this changes previously.

I have some interogations:

**Plugged into WP Dev API**
@frodrigo Here the API gives the route length in meters ?
![Image](https://github.com/user-attachments/assets/ce818a56-6da4-4d53-8674-866e6b40cf73)

**Plugged into Elasa Dev**
@frodrigo Here the API gives the route length in kilometers.
Plus, the translations are not working as I don't have any value associated with key trad.
![Image](https://github.com/user-attachments/assets/880f93ce-1f67-4b6a-9287-3a83a66e5a64)

@frodrigo As of today, getting kilometers from Elasa is fine as the formatting is expecting this format !
What about the translations ?

**Comment by frodrigo** [comment]
> I might have done this changes previously.

No. `11200.1` should by displayed using locale to smoothing like "11.200,1" using the french locale.

**Comment by frodrigo** [comment]
> So we keep receiving length in meters from API ??

The both API are OK. The example you find, contains invalid data.

> Do you agree on the fact that we need to define a unit format from the API ?

No.
